daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -428,6 +442,136 @@ execClean projectOpts = do
             removeAndWarn "dist"
             putStrLn "Removed build artifacts."
 
+
+
+data Action = ACreate (LF.Qualified LF.TypeConName) 
```

Comment:
I would prefer if most of the visualization logic could go into a separate module.